build.py

```diff
@@ -276,37 +287,45 @@ def do_dist_clean():
     print 'Removing 3rd-party libraries...'
     cmd('rm -rf %s %s' % (DPDK_FILE, DPDK_DIR))
 
-def print_usage():
-    print >> sys.stderr, \
-            'Usage: %s [all|dpdk|bess|kmod|clean|dist_clean|help]' % \
-            sys.argv[0]
+def print_usage(parser):
+    parser.print_help(file=sys.stderr)
     sys.exit(2)
 
+def update_benchmark_path(path):
+    print 'Specified benchmark path %s' % path
+    cxx_flags.extend(['-I', '%s/include'%(path)])
```

Comment:
just a minor nit: let's not add a whitespace after `-I` and `-L`, to be consistent with the existing `Makefile` style